findbugs/src/antTask/edu/umd/cs/findbugs/anttask/tasks.properties

```diff
@@ -1,5 +1,4 @@
-findbugs=edu.umd.cs.findbugs.anttask.FindBugsTask
-findBugs=edu.umd.cs.findbugs.anttask.FindBugsTask

```

Comment:
I think we should keep these 2 lines for the time being, in spite of adding the `spotbugs` definition. That way we can remain compatible with all other tools (maven plugin, gradle, etc.) that would only require to change the dependency from findbugs to spotbugs to keep on working. We will eventually drop them, but retaining them for the time being is probably for the best.